### PR TITLE
Implements WarLog and Player

### PR DESCRIPTION
**Pull request by azacchino**

### CODE REVIEW

**Comment by azacchino** [comment]
Implements WarLog, I tried this, and it works fine.
Regards

**Comment by azacchino** [comment]
@toniperic Hey, good day. You could see this?

File: src/Api/Player/Player.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      "trophies": 0,
+      "id": "string"
+    },
+    "bestSeason": {
+      "rank": 0,
+      "trophies": 0,
+      "id": "string"
+    }
+  },
+  */
```
Comment (toniperic) [inline src/Api/Player/Player.php]: Could you please get rid of this chunk of comments?

**Comment by toniperic** [comment]
@died74 hey, yeah, sorry for the late reply. Thanks for contributing. In some of the files you're using tabs, while on the others you're using spaces. Could you switch to spaces, so it's constant throughout the whole library? Thanks

File: src/Api/Player/Player.php (hunk 1 above)
Comment (azacchino) [reply src/Api/Player/Player.php]: Done.

This commenst are NOT implemented part of Player Response

**Comment by azacchino** [comment]
@toniperic  Hey, soryy for my poor English i'm speak spanish.

Ready, all files fixed.

File: src/Api/Player/AchievementList.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace ClashOfClans\Api\Player;
+
+use ClashOfClans\Api\AbstractResource;
+
+/**
+
```
Comment (toniperic) [inline src/Api/Player/AchievementList.php]: Could you also remove these empty comments?

Comment (azacchino) [reply src/Api/Player/AchievementList.php]: ready

**Comment by azacchino** [comment]
@toniperic done

**Comment by toniperic** [comment]
Thanks!

**Pull request merged**